lib/mongoose_push_web/controllers/api_v1_controller.ex

```diff
@@ -0,0 +1,34 @@
+defmodule MongoosePushWeb.APIv1Controller do
+  alias MongoosePushWeb.Schemas
+  alias OpenApiSpex.Operation
+  use MongoosePushWeb, :controller
+
+  @spec open_api_operation(atom) :: Operation.t()
+  def open_api_operation(action) do
+    operation = String.to_existing_atom("#{action}_operation")
+    apply(__MODULE__, operation, [])
+  end
+
+  @spec handle_operation() :: Operation.t()
+  def handle_operation() do
+    %Operation{
+      tags: ["apiv1"],
+      summary: "sends a push",
+      description: "performs the sending of push notification",
+      operationId: "APIv1Controller.handle",
+      parameters: [
+        Operation.parameter(:id, :path, :string, "Device ID", example: "f53453455", required: true)
+      ],
+      requestBody:
+        Operation.request_body(
+          "The push notification attributes",
+          "application/json",
+          Schemas.APIv1Request,
```

Comment:
Same thing as with `handle` operation - this is too generic name. This is `push` request for APIv1. Also, let's try to split the naming into more submodules, like `Schemas.APIv1.PushRequest`, or `Schemas.APIv1.Request.Push`.

Reply:
We need to figure out the structure here, as some input / output objects will be shared between API versions, and some will be completely different (APIv2 and APIv3 share entire request object, those differ only for response).

Reply:
I do agree on keeping things in order and preventing any chaos here, especially regarding future development, but maybe should we stick for now with current file tree and develop it altogether with further PRs, as it's going to grow? Or you'd like to have it already set up :)

Reply:
It's gonna be more work to refactor that later when we have more APIs defined. Also, I would argue that this change makes code more readable which affects the code even right now :)